src/routes/data.ts

```diff
@@ -4,6 +4,7 @@ export const ROUTES = {
   IMPORT: '/import',
   OVERVIEW: '/overview',
   ACCOUNTS: '/accounts',
+  ACCOUNT: '/account',
```

Comment:
For navigation `/accounts/:accountId` can be used, then it will be possible to use Link in a following way `<Link to={accountid} />`